package-lock.json

```diff
@@ -5204,7 +5204,7 @@
         "listr": "0.12.0",
         "lodash": "4.17.11",
         "log-symbols": "2.2.0",
-        "minimist": "1.2.5",
+        "minimist": "1.2.0",
```

Comment:
Can we remove `package-lock.json` changes ? This is updating `minimist` to vulnerable versions

Reply:
Yep, must have missed this. Resolved.